dev/restinio/ip_blocker.hpp

```diff
@@ -68,7 +69,7 @@ public :
 	//! Initializing constructor.
 	incoming_info_t(
 		endpoint_t remote_endpoint )
-		:	m_remote_endpoint{ remote_endpoint }
+		:	m_remote_endpoint{std::move( remote_endpoint )}
```

Comment:
I don't think the move-initialization for value-type `endpoint` is more efficient than simple copy-initialization.

Reply:
I thought that move-initialization is always more efficient than copy-initialization. At least this change, although with a negligible impact, was proposed by clang-tidy.

Reply:
> I thought that move-initialization is always more efficient than copy-initialization.

It's not true for small value-only types.